build.gradle

```diff
@@ -19,14 +19,21 @@ mainClassName = 'minicraft.core.Game'
 
 repositories {
     mavenCentral()
+    maven { url "https://plugins.gradle.org/m2/" }
+    maven { url "https://jitpack.io" }
 }
 
 dependencies {
     implementation 'org.jetbrains:annotations:23.0.0'
-    implementation 'org.json:json:20211205'
-    implementation 'com.konghq:unirest-java:3.13.7'
+    implementation 'org.json:json:20220320'
+    implementation 'com.konghq:unirest-java:3.13.10'
     implementation 'org.tinylog:tinylog-api:2.4.1'
     implementation 'org.tinylog:tinylog-impl:2.4.1'
+    implementation 'com.badlogicgames.gdx:gdx:1.11.0'
+    implementation 'com.badlogicgames.gdx:gdx-box2d:1.11.0'
+    implementation 'com.badlogicgames.gdx:gdx-controllers:1.9.13'
```

Comment:
Do we need these three when you have Jamepad installed?

Reply:
Yes, they are required for basic controller functions, with this jamepad and it's other dependency works fine.